cms/envs/acceptance.py

```diff
@@ -127,3 +127,7 @@ def seed():
 # Generate a random UUID so that different runs of acceptance tests don't break each other
 import uuid
 SECRET_KEY = uuid.uuid4().hex
+
+############################### PIPELINE #######################################
+
+PIPELINE_ENABLED = False
```

Comment:
Do you know why? Is it because all of the js/css is bundled, and those files take a long time to load? Should the lettuce tests be more resilient for wait times like this?